tests/acceptance/00_basics/macros/if_triple.cf

```diff
@@ -22,9 +22,10 @@ bundle agent init
 bundle agent check
 {
   methods:
-      "check" usebundle => dcs_passif_output(".*$(G.testdir)/test.cf Pass.*",
+      "check" usebundle => dcs_passif_output(regex_replace(".*$(G.testdir)$(const.dirsep)test.cf Pass.*",
+                                                           "\\\\", "\\\\\\\\", "g"),
```

Comment:
the amount of backslashes here kind of begs a comment I think. :)

Reply:
escape × escape backslash!